frontend/src/pages/InquiryOverview.js

```diff
@@ -1,19 +1,28 @@
-import {postInquiry} from "../services/inquiryService";
+import {getInquiries, postInquiry} from "../services/inquiryService";
 import { useEffect, useState } from 'react'
+import FormReactHookCreateNewInquiry from "../components/FormReactHookCreateNewInquiry";
+import InquiryList from "../components/InquiryList";
 
 export default function InquiryOverview(){
     const [inquiries, setInquiries] = useState([])
 
+    useEffect(()=>{
+        getInquiries()
+            .then(setInquiries)
+            .catch((error) => console.error(error))
+    })
+
     const addNewInquiry = (inquiry) => {
         postInquiry(inquiry)
             .then((newInquiry)=>{
                 const updatedInquiries = [...inquiries, newInquiry]
                 setInquiries(updatedInquiries);
-            })
+            }) .catch((error) => console.error(error))
     }
     return(
         <>
-
+            <InquiryList inquiries={inquiries}/>
```

Comment:
`{inquiries && <InquiryList inquiries={inquiries}/>}`
might make more sense to ensure list is only shown when inquiries are also available